examples/read-polygon-shapefile-and-convert-to-geo-types/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[package]
+name = "read-polygon-shapefile-and-convert-to-geo-types"
+version = "0.1.0"
+authors = ["Michel Kuhlmann <michel.kuhlmann@tkconsult.ch>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+shapefile = {version = "0.3.0", features = ["geo-types"]}
+dbase = "0.2.0"
```

Comment:
You actually don't need to add dbase as a dependency as its alreeady a shapefile dependency.
Remove it and in the main.rs add `use shapefile::dbase`